tests/rvt_test.cpp

```diff
@@ -101,12 +101,12 @@ class RVTTest
 };  // class
 
 /* Create instance of test class ---------------------------------------------------------- */
-RVTTest base;
+RVTTest BASE;
```

Comment:
why is BASE all caps? Is this a special case for gtest classes?

Reply:
I think it's because BASE is a global variable

Reply:
Ah, missed that. Thanks